model-optimizer/extensions/front/caffe/ctcgreedydecoder_ext_test.py

```diff
@@ -54,7 +54,7 @@ def test_ctcgreedydecoder_ext_ideal_numbers(self, merge_attrs_mock):
         exp_res = {
             'type': "CTCGreedyDecoder",
             'ctc_merge_repeated': 1,
-            'infer': CTCGreedyDecoderOp.ctc_greedy_decoder_infer
+            'infer': CTCGreedyDecoderOp.infer
```

Comment:
I have not refactored caffe part. I just had to change it due to reasonable refactoring the operation. If it is really needed, then it will be in a separate activity.